libs/components/flyout/src/lib/modules/flyout/flyout.service.ts

```diff
@@ -97,8 +94,6 @@ export class SkyFlyoutService implements OnDestroy {
             this._ngZone.onStable.pipe(take(1)).subscribe(() => {
               if (this.host) {
                 this.removeHostComponent();
-                // Without this tick - the host does not actually get removed on initial navigation in this case.
-                this.applicationRef.tick();
```

Comment:
Does the original PR/story that introduced this change give any insight as to _why_ we added this to begin with?